pkg/ebpf/c/bpf_telemetry.h

```diff
@@ -117,12 +117,12 @@ static void *(*bpf_telemetry_update_patch)(unsigned long, ...) = (void *)PATCH_T
     (__APPLY_OPx(x, __NEQ, __AND, __VA_ARGS__))
 
 #define __nth(_, _1, _2, _3, N, ...) N
-#define __nargs(...) __nth(_, ##__VA_ARGS__, 3, 2, 1, 0)
+#define nargs(...) __nth(_, ##__VA_ARGS__, 3, 2, 1, 0)
```

Comment:
This could cause conflicts if someone defines "nargs" as a variable in their programs, no?

Reply:
That should work, less likely to be used.